doc/news/changes.h

```diff
@@ -402,9 +402,16 @@ inconvenience this causes.
 <h3>Specific improvements</h3>
 
 <ol>
-  <li> New: A function to get a map with all vertices at boundaries
-  has been added at GridTools::get_all_vertices_at_boundary.
-  This function will return a map which can be used in functions
+  <li> New: There is now a new quadrature formula in quadrature_lib. It is
+  now possible to use Telles' quadrature rules to integrate singular
+  integrals
+  <br>
+  (Nicola Giuliani, 2015/04/01)
```

Comment:
Please reference `QTelles` somewhere. This way doxygen automatically links to it for people to click through for the full description.